modules/con-vuln-sources.adoc

```diff
@@ -0,0 +1,57 @@
+
+OVAL vulnerability data is more advisory based, while VEX data is more CVE based. One advisory can have multiple CVEs associated with it. Therefore, {product-title-short} can list different numbers of vulnerabilities when you are scanning with a {product-title-short} version that uses OVAL, such as {product-title-short} version 4.5, and a version that uses VEX, such as version 4.6. Additionally, VEX data does not include vulnerabilities with a status of "under investigation", while these vulnerabilities are included in OVAL data.
+
+For more information about Red Hat security data, including information about the use of OVAL, Common Security Advisory Framework Version 2.0 (CSAF), and VEX, see link:https://www.redhat.com/en/blog/future-red-hat-security-data[The future of Red Hat security data].
+
+//is this still applicable?
+The https://access.redhat.com/security/data/metrics/cvemap.xml[Red{nbsp}Hat CVE Map]: This is used for images which appear in the link:https://catalog.redhat.com/software/containers/explore[Red{nbsp}Hat Container Catalog].
+
+[id="other-vuln-sources_{context}"]
+== Additional vulnerability sources
+
+The following vulnerability sources are also used:
+
+* link:https://secdb.alpinelinux.org/[Alpine Security Database]
+* Data tracked in link:https://alas.aws.amazon.com/index.html[Amazon Linux Security Center]
+* link:https://security-tracker.debian.org/tracker/data/json[Debian Security Tracker]
+* link:https://linux.oracle.com/security/oval[Oracle OVAL]
+* link:https://packages.vmware.com/photon/photon_oval_definitions/[Photon OVAL]
+* link:https://support.novell.com/security/oval/[SUSE OVAL]
+* link:https://security-metadata.canonical.com/oval/[Ubuntu OVAL]
+* link:https://osv.dev/[OSV]: This is used for language-related vulnerabilities, such as Go, Java, Node.js (JavaScript), Python, and Ruby. This source might provide GitHub Security Advisory (GHSA) IDs rather than CVE numbers for vulnerabilities.
++
+[NOTE]
+====
+The {product-title-short} Scanner V4 also uses the OSV database available at link:https://osv.dev/[OSV.dev] under link:https://github.com/google/osv.dev/blob/master/LICENSE[this license].
+====
+* link:https://nvd.nist.gov/vuln/search[NVD]: This is used for various purposes such as filling in information gaps when vendors do not provide information. For example, Alpine does not provide a description, CVSS score, severity, or published date.
++
+[NOTE]
+====
+This product uses the NVD API but is not endorsed or certified by the NVD.
+====
+* link:https://github.com/stackrox/stackrox/blob/master/scanner/updater/manual/vulns.go[StackRox]: The upstream StackRox project maintains a set of vulnerabilities that might not be discovered due to data formatting from other sources or absence of data.
```

Comment:
a related example of how the data source changes between v2 and v4. this is just for v4. there is another link for v2. I know I never mentioned it before (it seemed like information overload), but since we are already making changes, I figure why not bring it up

Reply:
Can you send me the link for the v2 source?

Reply:
sent above